.github/workflows/build.yml

```diff
@@ -144,6 +146,16 @@ jobs:
           os: 'ubuntu-24.04'
           tls: 'quictls'
           systemcrypto: '-UseSystemOpenSSLCrypto'
+        # No openssl system crypto on ubuntu-22.04
+        - plat: linux
```

Comment:
I think we can remove the `-plat` here (and the one below) because the `-os` is descriptive enough.

Reply:
I guess you just copied from above (which also have the problem). So, it's not a requirement to fix this.